modules/photo/include/opencv2/photo.hpp

```diff
@@ -851,6 +851,45 @@ CV_EXPORTS_W void stylization(InputArray src, OutputArray dst, float sigma_s = 6
 
 //! @} photo_render
 
+//! @addtogroup layerModelBlending
+//! @{
+
+//! layerModelBlending algorithm flags
+enum
```

Comment:
It is better to use SnakeCase naming style for enum values to avoid conflicts with preprocessor definitions. `enum` should be named, so it can be referred in documentation and used in function signature instead of `int`.

Reply:
Yes, I should guarantee a unified code style.